TransactionService/docker_context/TransactionServiceApi.py

```diff
@@ -31,8 +32,10 @@ async def buyStock(request):
     res, err = validateRequest(request.json, buy_schema)
     if not res:
         return response.json(errorResult(err, request.json), status=400)
-
     data = request.json
+
+    data = app.config['serviceLogic'].buyStock(data['trans_num'], data['user_id'], data['stock_symbol'], data['amount'])
+
     return response.json(data)
 

```

Comment:
We seem to be missing the add POST endpoint as well as the validation schema

Reply:
added this post endpoint and schema